app/utils.py

```diff
@@ -52,7 +52,14 @@ def get_now_utc():
     @staticmethod
     def single_client_ip(request):
         if request['client_ip']:
-            single_ip = request['client_ip'].split(',', 1)[0]
+            client_ip = request['client_ip']
+            for character in string.whitespace + OBSCURE_WHITESPACE:
+                client_ip = client_ip.replace(character, '')
+            ip_validation_pattern = re.compile(r'^[0-9.,\s]*$')
+            if ip_validation_pattern.fullmatch(client_ip):
```

Comment:
consider this enhancement:
```
 if ip_validation_pattern.fullmatch(client_ip) and client_ip.count(',') > 1:
```
just in case the expected format coming is in different

Reply:
Added this in